src/WVCOModule.cpp

```diff
@@ -79,8 +79,8 @@ class WvcoPatcher
         }
         auto otherOutput = getOutput(otherModuleWidget, Comp::MAIN_OUTPUT);
 
-        assert(myFMPort->type == PortWidget::INPUT);
-        assert(otherOutput->type == PortWidget::OUTPUT);
+        assert(myFMPort->type == Port::INPUT);
+        assert(otherOutput->type == Port::OUTPUT);
```

Comment:
unrelated changes - when I compiled with ASSERT enabled this didn't compile. in V2.0 these enums moved from PortWidget to Port. (no, not listed either).